NSS-Day-Cohort-66/rare-api-rare-poultry-api
# Comments View

# Comments View Updated to Reflect Client Side Needs

## The API should return:
1. The Title of the Post
2. comment context
3. date created
4. full name of the author

## Steps to Test:
1. git fetch
2. checkout -b PMComments
3. run debugger
4. Make a get request in postman to http://localhost:8000/comments
5. The return should meet all requirements above
